docs/site/Decorators_route.md

```diff
@@ -315,5 +315,5 @@ class MyController {
 ```
```

Comment:
Since you are here, can we rename `Decorators_route.md` to `Decorators_openapi.md`?

Reply:
sure. Do you want to rename the title to `OpenAPI Decorators` instead of `Route Decorators`?